fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -255,7 +279,24 @@ public PlanFragment visitPhysicalSort(PhysicalUnaryPlan<PhysicalHeapSort, Plan>
         return mergeFragment;
     }
 
-    // TODO: 1. support shuffle join / co-locate / bucket shuffle join later
+    /**
+     * Populate the SortTupleSlotExprs member.
+     */
+    private void createSortTupleSlotExprs(SortInfo info,
+            TupleDescriptor tupleDescriptor,
+            List<Expr> outputList, List<Expr> orderingExpr) {
+        List<Expr> sortTupleSlotExprs = new ArrayList<>();
+        Set<ExprId> exprIds = new HashSet<>();
+        outputList.addAll(orderingExpr);
+        for (Expr expr : outputList) {
+            if (!exprIds.contains(expr)) {
```

Comment:
exprIds never update. and use ExprId compare with Expr

Reply:
done